### PR TITLE
BZ-1986945: Explain confirmation of template availability

### PR DESCRIPTION
**Pull request by lmandavi**
This GitHub issue is associated with https://issues.redhat.com/browse/CNV-13235 and https://bugzilla.redhat.com/show_bug.cgi?id=1986945.

### CODE REVIEW

**Comment by lmandavi** [comment]
@yaacov  - Requesting SME review. Thanks!

**Comment by lmandavi** [comment]
https://deploy-preview-35511--osdocs.netlify.app/openshift-enterprise/latest/virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.html#virt-importing-rhel-image-boot-source-web_virt-creating-and-using-boot-sources

**Comment by lmandavi** [comment]
@yaacov - Requesting SME review. Thank you!

**Review by yaacov — APPROVED** [review]
LGTM

**Comment by lmandavi** [comment]
@gouyang - Requesting QE review. Thank you!

**Comment by lmandavi** [comment]
@demilyc - It is "tile" and not "title". Each Red Hat template displays as a tile in the web console. These templates display in the form of tiles. Therefore, they are tiles and not titles. Thank you for your review.

**Comment by lmandavi** [comment]
@demilyc Requesting QE approval so that this PR can move to peer review.

**Comment by demilyc** [comment]
LGTM

**Comment by lmandavi** [comment]
Requesting peer review.

File: modules/virt-importing-rhel-image-boot-source-web.adoc [hunk 1]
```diff
@@ -27,4 +27,5 @@ You can import a Red Hat Enterprise Linux (RHEL) image as a boot source by speci
 . In the *Add boot source to template* window, paste the copied URL of the guest image into the *Import URL* field, and click *Save and import*.
 
 .Verification
-* In the *Templates* tab, verify that the *Boot source* column displays *Available* for the RHEL virtual machine template. You can use this template to create RHEL virtual machines.
+
+* In the *Templates* tab, verify that the tile for this template displays a green checkmark to confirm that this template already has a boot source added. You can use this template to create RHEL virtual machines.
```
Comment (ousleyp) [inline modules/virt-importing-rhel-image-boot-source-web.adoc:31]: To make this easier to read, I'd suggest breaking up this step into shorter bits. Suggested rewrite:

```
To verify that a boot source was added to the template:

. Click the *Templates* tab.

. Confirm that the tile for this template displays a green checkmark. 

You can now use this template to create RHEL virtual machines.
```

What do you think?

**Review by ousleyp — COMMENTED** [review]
I have a suggestion for you to consider, but this mostly LGTM. Please also edit the PR description so that it contains the relevant branches for cherrypicking. Thanks, Lavanya!

File: modules/virt-importing-rhel-image-boot-source-web.adoc (hunk 1 above)
Comment (lmandavi) [reply modules/virt-importing-rhel-image-boot-source-web.adoc:31]: Makes it cleaner! Removes the clumping! Updated the content.

**Comment by lmandavi** [comment]
@ousleyp - This is ready for review. The content applies to CNV 4.8 and CNV 4.9 releases. Thank you!

**Pull request merged**